src/index.html

```diff
@@ -6,13 +6,65 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <title>Document</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="Product logo"
+      />
+
+      <div class="product-card__name">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="product-card__code">Product code: 195434</div>
+
+      <div class="product-card__info">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
```

Comment:
Ensure that the `stars` block is implemented in a way that it can be easily reused and can display different number of active stars based on some data attribute or CSS class.